common/build/eslint-config-fluid/base.js

```diff
@@ -102,9 +102,10 @@ module.exports = {
 				avoidEscape: true,
 			},
 		],
+		"@typescript-eslint/require-await": "off",
 		"@typescript-eslint/restrict-plus-operands": "error",
 		"@typescript-eslint/restrict-template-expressions": "off",
-		"@typescript-eslint/require-await": "off",
+		"@typescript-eslint/return-await": "error",
```

Comment:
(Post-merge comment) I think we should consider changing the default configuration for this one from `in-try-catch` to `always`. The stack trace benefit is worth it, IMO.